_data/tool_and_resource_list.yml

```diff
@@ -1,3 +1,16 @@
+# List of tools and resources
+#
+# When referencing tools and resources use the following notion e.g. on Task pages -  {% tool "github" %} and not a markdown link
+# for this to work the tool has to be listed here.
+#
+# - description: A short description of the purpose of the tool or resource
+#   id: This is the id by which the tool will be references on other pages e.g. for GitHub the id is github so we use {% tool "github"%} to refer to it
+#   name: This is the name of the tool or resource email: email adress
+#   registry: This is a list of registry:search-string combinations 
```

Comment:
I suggest to have a URL here for now.
```suggestion
#   registry: This is a list of registries where the tool is available 
```

Reply:
No - this is literally the list of 'registry:seach-string' combinations - which are searched to further information about this tool entry - I will make it clearer but not adding this suggestion.

Reply:
oh, I misunderstood then